ores/wsgi/util.py

```diff
@@ -61,3 +63,18 @@ def format_output(context, scores, model_info, warning=None, notice=None):
         for rev_id in scores[model]:
             output['scores'][context][model]['scores'][rev_id] = scores[model][rev_id]
     return jsonify(output)
+
+
+# Parse values for features / datasources of interest.
```

Comment:
Ok, but there are number of other PEP8 warnings in the library. Should they be fixed? Should CI not pass until they are?